### ISSUE TITLE
Fatal errors in widgets, because Twig. ("Defining the initRuntime() method…")

### ISSUE THREAD

**Issue by bobdenotter** [bolt/bolt#4450]
Twig is pooping out errors for deferred widgets:

```
Defining the getGlobals() method in the "Bolt" extension is deprecated without 
explicitly implementing Twig_Extension_GlobalsInterface.
```

and 

```
Defining the initRuntime() method in the "form" extension is deprecated. Use the 
`needs_environment` option to get the Twig_Environment instance in filters, functions, 
or tests; or explicitly implement Twig_Extension_InitRuntimeInterface if needed (not 
recommended).
```

![screenshot 2015-11-15 11 34 14](https://cloud.githubusercontent.com/assets/1833361/11168378/0c620262-8b8d-11e5-9597-150dcccc2fbf.png)

![screenshot 2015-11-15 11 34 22](https://cloud.githubusercontent.com/assets/1833361/11168379/0df26018-8b8d-11e5-92b7-6db6afc344c3.png)

Seems to be because of this: https://github.com/twigphp/Twig/blob/fd84403ac725fe1a053a0a109e26ecdb9928c912/CHANGELOG#L5

Related: https://github.com/kriswallsmith/assetic/issues/768

**Comment by bobdenotter** [comment]
PR for first part of the fix coming in. 

The second part needs to be fixed in Twig_Bridge. It's already in, but won't be availably until 2.7.7 is tagged. 

https://github.com/symfony/twig-bridge/blob/2.7/Extension/FormExtension.php#L24

**Comment by bobdenotter** [comment]
Second part is fixed by #4468
